src/platforms/common/configuration/options.mdx

```diff
@@ -126,6 +126,18 @@ This parameter controls if integrations should capture HTTP request bodies. It c
 
 </ConfigKey>
 
+<ConfigKey name="max-request-body-size" supported={["php"]}>
+
+This parameter controls if integrations should capture HTTP request bodies. It can be set to one of the following values:
+
+- `never`: request bodies are never sent
```

Comment:
Thanks, but as already pointed out, PHP uses `none` instead of `never`. Xref https://github.com/getsentry/sentry-php/issues/1029